### PR TITLE
Molecule Text created

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
Molecule Text created

### CODE REVIEW

File: stories/Molecules/Text/Faq/Faq.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+         <div className="arrow down"></div>
+      </div>
+      <div className="accordion-body">
+         <P label={descriptionText} />
+         <P label={descriptionText} />
+         <P label={descriptionText} />
+      </div>
+      <div className="accordion-head">
+         <H5 label={headerText} />
+         <div className="arrow down"></div>
```
Comment (surbhi90) [inline stories/Molecules/Text/Faq/Faq.jsx:25]: @harishkumar-srijan Please check if we can use the arrow in heading :after

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Text/BlockCTA/blockCTA.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @include devicebreak(large) {
+      margin-bottom: $spacing-48;
+    }
+  }
+  h5 {
+    margin-bottom: $spacing-12;
+    @include devicebreak(large) {
+      margin-bottom: $spacing-24;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Molecules/Text/BlockCTA/blockCTA.scss]: remove H5 , done

File: stories/Molecules/Text/Faq/Faq.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+
+/* Accordian start */
+.accordion {
+  width: 100%;
+  .accordion-head {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Molecules/Text/Faq/Faq.scss]: Done

File: stories/Molecules/Text/Faq/Faq.jsx (hunk 1 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Text/Faq/Faq.jsx]: Done

File: stories/Molecules/Text/Faq/Faq_custom.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      $(this).removeClass('open').addClass('close');
+      $accordian.find('.accordion-body').slideUp();
+      if (!$(this).next().is(':visible')) {
+        $(this).removeClass('close').addClass('open');
+        $(this).next().slideDown();
+      }
+    });
+  });
+}
+/* Faq JS end custom */
```
Comment (surbhi90) [inline stories/Molecules/Text/Faq/Faq_custom.js:16]: @harishkumar-srijan We have to place all the js files in assets/js folder

File: stories/Molecules/Text/Faq/Faq.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+</Story>
+
+<div className="subheading">Dependency</div>
+
+Libraries or another dependency which are used in the component.
+
+<ul>
+<li>For toggle functionality its depend on the jQuery plugin library, that is placed in preview-head.html file.</li>
+<li>Custom JS file (Faq.js) is located in the same component folder.</li>
+<li>ID - (#accordion) is used in the JSX file to connect the Custom JS file to achive the functionality.</li>
```
Comment (surbhi90) [inline stories/Molecules/Text/Faq/Faq.stories.mdx:61]: @harishkumar-srijan correct the achieve spelling

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Text/Faq/Faq.stories.mdx (hunk 5 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Text/Faq/Faq.stories.mdx]: Done

File: stories/Molecules/Text/Faq/Faq_custom.js (hunk 4 above)
Comment (harishkumar-srijan) [reply stories/Molecules/Text/Faq/Faq_custom.js]: Done

**Review by surbhi90 — APPROVED** [review]
LGTM

**Pull request merged**